### DISCUSSION TITLE
Migrating solid unit tests to op

### DISCUSSION THREAD

**Discussion by ivanov-slk** [dagster-io/dagster#5654] [Q&A] [answered]
As of version 0.12 I invoke solids in unit tests roughly like:
```
result = execute_solid(my_solid,
                               input_values={'inp1': inp1},
                               run_config={
                                   'solids': {
                                       'my_solid': {
                                           'config': {
                                               'arg1': arg1,
                                               'arg2': arg2
                                           }
                                       }
                                   }
                               })
```
and then compare `result.output_value()` with an expected value.

Given that in v0.13 `op` has replaced `solid`, how am I supposed to execute unit tests with a configuration, like above? The only thing I found in the [documentation](https://docs.dagster.io/tutorial/intro-tutorial/testable#testing-the-cereal-job-and-its-ops) mentiones only that `op`s can be invoked like a regular Python function. How am I supposed to supply them with `context`, as I could in v0.12 and before?

Thanks!

**Comment by sryza** [comment, marked as answer]
Hi @ivanov-slk - here's documentation on how to supply ops with context objects when invoking them directly: https://docs.dagster.io/concepts/testing#testing-ops.

Let us know if you run into any issues.

**Reply by ivanov-slk** [reply]
Thank you, I missed that part of the documentation. :/
